app/javascript/application.js

```diff
@@ -3,9 +3,14 @@ import "@hotwired/turbo-rails"
 import "./controllers"
 import React from 'react';
 import ReactDOM from 'react-dom';
+import Greetings from "./components/Greetings";
 
 function App() {
-  return (<h1>Hello World!!!</h1>);
+  return (
+    <div>
+      <Greetings />
+  </div>
```

Comment:
- [ ] It would be better if our `App` component has a route that leads to the `Greeting` component. Currently, we are missing the routes